templates/test/module_table.html.ep

```diff
@@ -49,7 +49,11 @@
                             % my $href  = "#step/$module->{name}/$step->{num}";
                             % if ($is_parser_text_result) {
                                 <span title="<%= $title %>" data-href="<%= $href %>" onclick="toggleTextPreview(this)" class="text-result">
-                                    <span class="resborder <%= $resborder %>"><%= $step->{text_data} %></span>
+                                    <span class="resborder <%= $resborder %>"><%= $step->{text_data} %><span class="step_actions" style="float: right; white-space: normal;">
```

Comment:
please but the inner `<span>` into a new indented line, the closing `</span>` accordingly. It is ok to test this style changes here but to have it merge-able I recommend to put that value for style in an according CSS style file. You can probably experiment with this more easily within your browser developer tools. Finally this should probably go into assets/stylesheets/test-details.scss . The look is ok for me though.

@Martchus do you have an idea to answer the question regarding putting the icon top-right w/o increasing the row size?

Reply:
> please but the inner `<span>` into a new indented line, the closing `</span>` accordingly.

I've tried, but since this span has `white-space: pre-wrap`, it will cause that newline to actually affect the visual effect. That's why `<%= $step->{text_data} %>` is on the same line here too.

Reply:
I needed to checkout myself. But `style="float: right; white-space: normal;` should better go into its own CSS class.

> I've tried, but since this span has white-space: pre-wrap

Yes, without changing the HTML/CSS more (e.g. having only the `text_data` text within its own span) it is not possible to change this. I don't think it is important, though.